tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml

```diff
@@ -157,6 +157,13 @@ SpatialDiscretization:
     Formulation: StrongInertial
     Quadrature: GaussLobatto
 
+Filtering:
+  ExpFilter0:
+    Alpha: 36.0
+    HalfPower: 210
```

Comment:
this seems a bit extreme...

Reply:
Do you have a value you'd prefer? Note the filter is disabled by default anyway, and in real life the user will almost certainly need to change this depending on how they change the resolution

Reply:
how about 24 or 32...